### PR TITLE
Requesting Subdomain

### PR DESCRIPTION
**Pull request by satr14washere**
Hi, Im requesting the subdomain `sx9.is-a.dev` for my personal website!

### CODE REVIEW

File: domains/sx9.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{ 
+  "description": "SX's Website", 
+  "repo": "https://sx-9.github.io", 
+  "owner": { 
+    "username": "SX-9", 
+    "twitter": "SX_Discord", 
+    "emails": {
```
Comment (MaskDuck) [inline domains/sx9.json:7]: only use 1 email. thanks.

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Review by MaskDuck — APPROVED** [review]
gud now

**Comment by SX-9** [comment]
Thx

**Comment by SX-9** [comment]
Uh...

File: domains/sx9.json (hunk 1 above)
Comment (SX-9) [reply domains/sx9.json:8]: Use my outlook email

**Pull request merged**